openfec/entities/annotate.py

```diff
@@ -1,132 +0,0 @@
-"""
-Tools for annotating records
```

Comment:
I took these out because I thought we had already removed them from master and that I was accidentally adding things back in. This was Sean's earlier work. I can put it back in if you want, @arowla .